__tests__/commands/info.js

```diff
@@ -42,16 +42,18 @@ const expectedKeys = [
   'bugs',
   'license',
   'scripts',
-  'dependencies',
-  'devDependencies',
   'dist',
   'directories',
 ];
 
+// yarn now ships as built, single JS files so it has no dependencies
```

Comment:
Yeah. I should go over most of the tests including git ones and make everything offline if possible leaving very few actual e2e tests.